gpio/zynq7000-gpio/gpio.c

```diff
@@ -0,0 +1,296 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Zynq-7000 GPIO controller
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Lukasz Kosinski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <errno.h>
+#include <stdlib.h>
+
+#include <sys/mman.h>
+#include <sys/platform.h>
+
+#include <phoenix/arch/zynq7000.h>
+#include <board_config.h>
+
+#include "gpio.h"
+
+
+/* GPIO registers */
+enum {
+	dlsw0 = 0, dmsw0, dlsw1, dmsw1,
+	data0 = 16, data1,
+	rodata0 = 24, rodata1,
+	dirm0 = 129, oen0, imr0, ier0, idr0, isr0, ityp0, ipol0, iany0,
+	dirm1 = 145, oen1, imr1, ier1, idr1, isr1, ityp1, ipol1, iany1,
+};
+
+
+/* GPIO pins */
+const int gpioPins[GPIO_BANKS][GPIO_PINS] = {
```

Comment:
```suggestion
static const int gpioPins[GPIO_BANKS][GPIO_PINS] = {
```

Reply:
I want `gpioPins` to be visible in `gpiosrv.c` (it's used for creating device files for the configured pins). This is why I export this array in `gpio.h`.